rules/variables.js

```diff
@@ -0,0 +1,17 @@
+// Variables http://eslint.org/docs/rules/#variables
+
+module.exports = {
+	"rules": {
+		"init-declarations": [ 1, "always" ], //  enforce or disallow variable initializations at definition
```

Comment:
Discuss, do we want this to be ignored (0) or to be an error (2)?

http://eslint.org/docs/rules/init-declarations.html